bindata/network/ovn-kubernetes/ipsec.yaml

```diff
@@ -238,10 +238,20 @@ spec:
             - -c
             - |
               #!/bin/bash
-              ovs-appctl -t ovs-monitor-ipsec ipsec/status
+              ovs-appctl -t ovs-monitor-ipsec ipsec/status && ipsec status
```

Comment:
Was this broken before?

Reply:
Not broken but this is an improvement. I will split this into a separate commit to make it clear. Thanks for the review.